config/vufind/KohaRest.ini

```diff
@@ -177,3 +177,10 @@ extraFields = item-issue:acceptTerms:pickUpLocation
 ; to disable it:
 ;purgeAll = false
 
+; This section allows libraries to define custom renewal block messages for specific item types.
+; The first string is the item type. In the below example, ILL.
+; The second string after the equal sign is the Koha returned renewal block reason
+; A colon delineates it from the renewal block message to display in VuFind.
```

Comment:
This comment block needs to be updated to reflect the changes in the actual settings.